saleor/graphql/attribute/utils.py

```diff
@@ -1371,3 +1376,23 @@ def prepare_error_list_from_error_attribute_mapping(
         errors.append(error)
 
     return errors
+
+
+def prepare_permission_text_for_description(permissions):
+    return " or ".join([f"`{perm}`" for perm in permissions])
+
+
+def _validate_permissions_for_attribute(context, permissions):
+    if permissions and not has_one_of_permissions(
+        get_user_or_app_from_context(context), permissions
+    ):
+        raise PermissionDenied(
+            message=message_one_of_permissions_required(permissions).lstrip("\n")
+        )
+
+
+def check_permissions_for_attribute(context, attribute, permission_map):
+    if permissions := permission_map.get(attribute.type):
+        _validate_permissions_for_attribute(context, permissions)
+    else:
+        raise RuntimeError(f"Unknown attribute type: {attribute.type}")
```

Comment:
Can you explain, when this can happen?

Reply:
This could happen when type of attribute will be different that page type or product type.